esx_service/utils/auth_api.py

```diff
@@ -398,36 +410,57 @@ def _tenant_ls(name=None):
     error_info, tenant_list = get_tenant_list_from_db(name)
     return error_info, tenant_list
 
-def any_vm_already_exist(existing_vms, vms):
+def vm_already_exist(name, vms):
```

Comment:
The name of this method and the "vm_in_tenant" method below may cause some confusion. To clarify the purpose of these two methods, I'd suggest to make the following changes:
1. vm_already_exist(name, vms) => vm_already_in_tenant(name, vms)
2. vm_in_tenant(name, vms) => vm_in_any_tenants(vms)

Please note that I have removed the "name" parameter from the 2nd method because we don't really need the tenant name.